modules/local/process/merge_bam.nf

```diff
@@ -4,14 +4,16 @@ process MERGE_BAM {
     tag "${meta.id}"
 
     input:
-        tuple val(meta), path(bam), path(bai)
+        tuple val(meta), path(bam)//, path(bai) optional: true
```

Comment:
`optional: true` for the input?

Reply:
This is void now, sorry still working on this.